CHANGELOG.md

```diff
@@ -37,6 +37,15 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
   - Removed obsolete `TestKit::blockchain_mut` method and `TestKit::blockchain`
   now returns value instead of reference.
 
+- `system/v1/proto-sources` endpoint has been divided into two endpoints. (#1511)
```

Comment:
I think placeholder should be kept the most-bottom one entry until it's replaced with actual changes description.